src/ApiLoader/EnableInTracee.cpp

```diff
@@ -11,6 +11,7 @@
 #include <functional>
 
 #include "ObjectUtils/LinuxMap.h"
+#include "OrbitBase/Align.h"
```

Comment:
Should this be replaced by `ObjectUtils/LinuxMap.h`?

Reply:
done.